tests/test_connection.py

```diff
@@ -15,14 +23,15 @@
 
 AddrIp = ["127.0.0.1", "::1"]
 port = 9669
+handshakeKey = "3.0.0"
```

Comment:
please add some tests for wrong handshakeKey value, such as "INVALID_HANDSHAKE_KEY"